### PR TITLE
Phc module updated 

### PR DESCRIPTION
**Pull request by krishnanjali25**

### CODE REVIEW

**Review by arshadsid — APPROVED** [review]
Changes in models are valid.

File: FusionIIIT/applications/health_center/views.py [hunk 1]
```diff
@@ -1,7 +1,223 @@
...
+from .models import (Ambulance_request, Appointment, Complaint, Constants,
+                     Doctor, Hospital_admit, Medicine, Prescribed_medicine,
+                     Prescription, Stock, Stockinventory)
+
 
+@login_required
 def healthcenter(request):
-    context = {}
+    usertype = ExtraInfo.objects.get(user=request.user).user_type
+    if(usertype == 'student'):
```
Comment (Samy-33) [inline FusionIIIT/applications/health_center/views.py]: @krishnanjali25 remove these unnecessary parenthesis in `if` statement.

File: FusionIIIT/applications/health_center/views.py [hunk 2]
```diff
@@ -1,7 +1,223 @@
...
+        return HttpResponseRedirect("/healthcenter/student")
+    elif (usertype == 'compounder'):
```
Comment (Samy-33) [inline FusionIIIT/applications/health_center/views.py]: same

File: FusionIIIT/applications/health_center/views.py [hunk 3]
```diff
@@ -1,7 +1,223 @@
...
+        return HttpResponseRedirect("/healthcenter/compounder")
+
+
+def compounder_view(request):
```
Comment (Samy-33) [inline FusionIIIT/applications/health_center/views.py]: It will be great if you redesign this view.

**Pull request closed without merging**